internal/app/tfsec/checks/aws047.go

```diff
@@ -0,0 +1,93 @@
+package checks
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/tfsec/tfsec/internal/app/tfsec/parser"
+	"github.com/tfsec/tfsec/internal/app/tfsec/scanner"
+)
+
+const AWSSqsPolicyWildcardActions scanner.RuleCode = "AWS047"
+const AWSSqsPolicyWildcardActionsDescription scanner.RuleSummary = "AWS SQS policy document has wildcard action statement."
+const AWSSqsPolicyWildcardActionsExplanation = `
+SQS Policy actions should always be restricted to a specific set.
+This ensures that the queue itself cannot be modified or deleted, and prevents
```

Comment:
This is going to be rendered in the docs so doesn't need to be wrapped.

Can this be a new paragraph starting at `This` as well please.